utils/docker/vcluster/README.md

```diff
@@ -1,203 +1,4 @@
-# Using DAOS with Docker
+Docker recipes and scripts to build and deploy Docker images to simulate
+a small cluster using DAOS as backend storage.
```

Comment:
Maybe a reference of the page containing the full documentation could be helpful.

Reply:
See the line below?

Reply:
My bad